mmx_sale_triple_price/README.rst

```diff
@@ -0,0 +1,98 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Besides be careful of the comments in original PR.
Change name to sale_triple_price